rust/src/database/migrate/mod.rs

```diff
@@ -0,0 +1,71 @@
+use std::path::{Path, PathBuf};
+use crate::cli::ServerConfig;
+use anyhow::Result;
+use log::info;
+use scylla::Session;
+use crate::database::new_session;
+
+pub async fn migrate(server_config: &ServerConfig, should_drop_keyspace: bool) -> Result<()>
+{
+    let session = new_session(server_config).await.unwrap();
+
+    if should_drop_keyspace {
+        drop_keyspace(&session, server_config.keyspace.as_str()).await;
+    }
+
+    create_keyspace(&session, &server_config.keyspace.as_str()).await;
+    create_tables(&session).await;
+
+    Ok(())
+}
+
+async fn create_tables(session: &Session) {
+    let current_path = std::env::current_dir().unwrap();
+    let file_path = Path::new("src/database/migrate/migrate.cql");
+    let full_path = current_path.join(file_path);
+
+    let raw_queries = read_file(full_path)
+        .await
+        .unwrap_or_else(|_| panic!("Could not read file"));
+
+    let queries = raw_queries.split(";")
+        .map(|query| query.trim())
+        .collect::<Vec<&str>>();
+
+
+    info!("Creating tables...");
+    for query in queries {
+        if query.is_empty() {
+            continue;
+        }
+
+        let prepared_query = session.prepare(query).await.unwrap();
+        session.execute(&prepared_query, []).await.unwrap();
+    }
+    info!("Migration completed!");
+}
+
+async fn drop_keyspace(session: &Session, keyspace: &str) {
+    let query = format!("DROP KEYSPACE IF EXISTS {}", keyspace);
+    let prepared_drop = session.prepare(query).await.unwrap();
+    session.execute(&prepared_drop, []).await.unwrap();
```

Comment:
Same here. Unless you plan to hold you `PreparedStatement`s somewhere OR execute them multiple times, preparing them gives no benefit.